index.json

```diff
@@ -373,6 +373,23 @@
             "supports": "2.5.0"
         }
     },
+    "ucv-ext-control": {
+        "name": "Control",
+        "url": "https://urbancode.github.io/IBM-UCx-PLUGIN-DOCS/UCV/ucv-ext-control",
+        "description": "Sync pull request, commit and issues from Control",
+        "author": {
+            "name": "UrbanCode",
+            "email": "urbancode-plugins@hcl.com"
+        },
+        "current": {
+            "semver": "1.0.2",
+            "date": "2024-10-21T10:29:52.192Z",
+            "image": "urbancode/ucv-ext-control:1.0.2",
+            "notes": [
+                "Initial Release for Control Plugin"
+            ]
```

Comment:
you should add minimum supported velocity version always here.
"supports": "5.0.9" or choose with what version we are releasing it.